tools/webpack/blocks.js

```diff
@@ -30,6 +30,7 @@ const blockViewRegex = new RegExp(
 const prefixFunctions = [
 	'build_query_vars_from_query_block',
 	'wp_enqueue_block_support_styles',
+	'wp_style_engine_get_styles',
```

Comment:
Do we need this? Looks like `wp_style_engine_get_styles` is readily available as part of GB here: https://github.com/WordPress/gutenberg/blob/5fbd554da9ca8d23fca605d859a737cbb8c18513/packages/style-engine/style-engine.php#L41

Reply:
If I remove that, fatal error appears 🤔

Reply:
I'll have a look what might be going on there.

Reply:
I've tested the GH-generated plugin [artifact](https://github.com/WordPress/gutenberg/suites/8360750848/artifacts/368517412) against Core `trunk`. I'm not seeing a fatal 🤔 

How did you test?

Reply:
On the home page, after building (in the branch, WP Core trunk), I had it without that line. Call to undefined function.
![Screenshot 2022-09-19 at 16 44 37](https://user-images.githubusercontent.com/37012961/191059088-6ba6f907-e6a3-453c-b696-92a31f50b5f7.jpg)

But my local environment has been tricky today 😢

Reply:
Ah, this is the GB test env, right? Did you reset your `wp-env`? Maybe your local `wp-env` is outdated an hasn't picked up the latest Core `trunk` yet?

Reply:
🤦‍♂️ Disregard. Turns out I was testing the wrong plugin .zip -- the one that was built using the webpack rule.

Without it, I'm getting the fatal 👍